powerdnsadmin/routes/admin.py

```diff
@@ -763,7 +763,7 @@ def __init__(self, history, change_set):
             self.detailed_msg = ""
             return
 
-        detail_dict = json.loads(history.detail)
+        detail_dict = json.loads(history.detail.replace("'", '"'))
```

Comment:
I believe this one should not be reverted, but we should push an alembic script to replace quotes in all history entries instead.